01 - Community/People/javalent.md

```diff
@@ -1,25 +1,29 @@
 ---
 aliases:
 - Jeremy Valentine
+- Javalantasy
 tags:
-- 
+- incubator
 publish: true
 ---
 
-# Jeremy Valentine
+# Javalent
 
-- GitHub: [javalent](https://github.com/javalent/) ^github
-<!-- - Discord: `@` ^discord-->
-- Website: <https://github.com/valentine195> ^website
-<!-- - [[Publish sites|Publish site]]: <https://> ^publish-->
+- GitHub: [Javalent](https://github.com/javalent/) ^github
+- Discord: `@javalent#3452` ^discord
+- Website: <https://github.com/javalent> ^website
+- [[Publish sites|Publish site]]: <https://plugins.javalent.com/home> ^publish
 
 %% Feel free to add a bio below this comment %%
 
+>[!info] This is an duplicate person entry of [[valentine195|Jeremy Valentine]].
```

Comment:
'an duplicate' -> 'a duplicate'.

I found this confusing, as this and the other equivalent callout in the other file both say that they link to 'Jeremy Valentine', so it's unclear what the difference is.

But really, what's the thinking behind maintaining two files with largely duplicate content?

Reply:
I will correct the callout. 

I suppose I am not sure which to make the primary. Javalent the person organisation, or Javalent as Jeremy Valentine. I can remove the bulk of content of one easily enough. Is there any preference on which should be primary?

Reply:
Ended up making Javalent the primary as everything will eventually show being published by that org.